File: typescript/src/server/email/restApiEmailValidateResponse.ts
import { RestApiResponse } from "../restApiResponse";
import { RestApiResponseError } from "../restApiResponseError";
import { RestApiEmailMetadata } from "./restApiEmailMetadata";
import { RestApiEmailValidateResult } from "./restApiEmailValidateResult";

export type RestApiEmailValidateResponse = RestApiResponse & {
    error?: RestApiResponseError;
    result?: RestApiEmailValidateResult;
    metadata?: RestApiEmailMetadata;
};

File: csharp/DVSClient/DVSClient/server/email/RestApiEmailValidateResponse.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Email
{
    public class RestApiEmailValidateResponse : RestApiResponse
    {
        [JsonProperty("error")]
        public RestApiResponseError? Error { get; set; }

        [JsonProperty("result")]
        public RestApiEmailValidateResult? Result { get; set; }

        [JsonProperty("metadata")]
        public RestApiEmailMetadata? Metadata { get; set; }
    }
}
